uaclient/apt.py

```diff
@@ -72,6 +83,8 @@
 event = event_logger.get_event_logger()
 LOG = logging.getLogger(util.replace_top_level_logger_name(__name__))
 
+apt_pkg.init()
```

Comment:
Can't we have problems with that approach ? Every module that import `apt` will run the init command which can affect the apt cache right ?

Reply:
We need to init it at least once, for functionality that is not tied directly to the caches.
It should not affect the apt caches because we are initialize it when creating those, and encapsulate in PreserveAptConfig.

But I see we can use a guard there, to run it only once, and don't `init` again if it is already initializated

Reply:
So I have updated this with a guard statement - this should now only have an impact if the module is not yet initialized